core/archive_node.supervisord.conf

```diff
@@ -7,7 +7,7 @@ logfile=/dev/stdout
 logfile_maxbytes=0
 
 [program:archive_node]
-command=/usr/bin/stream_node archive
+command=/usr/bin/river_node archive
```

Comment:
we neither have code, build, nor run chain nodes ourselves though, I don't see how it can be confusing. this name better reflects multi-functional aspect of the binary.

Reply:
We should pick a name and stick with it. While I agree that we don't have any code for the chain nodes, everyone that we talk to, all of our documentation, and everyone that will join in the future, will see "river node" and think "block chain". I think we should call it the stream node - just in this pr you have run_stream_cmd, run_xchain_cmd, but you've got `river_node` and `xchain` folders... these should match up right?